### ISSUE TITLE
I cannot connect locally to the server.

### ISSUE THREAD

**Issue by itsjohnn22** [mschnitzer/ark-survival-ascended-linux-container-image#38]
I keep having an issue connection to the server locally, it keeps giving me connection timed out error. Server shows up on the server list. Any ideas why this keeps happening?

**Comment by mschnitzer** [comment]
Try to connect using the `open` command in ARK and try to connect using the local IP of the container. The command in ARK would look like this:

```
open $ip:7777
```

The IP of the container can be figured out by using this command:
```
docker inspect asa-server-1 | grep IPv4Address
```

Are you running the ASA client on Linux and thus on the same system as the game server container?

**Comment by itsjohnn22** [comment]
Yes, I've tried it on the same system through a VM. I also attempted it on my other rig. When I ran the open command in Ark, I encountered a black screen. When I try to run `docker inspect asa-server-1 | grep IPv4Address` in the terminal, I don't get any output.

**Comment by mschnitzer** [comment]
Could you please post the full output of `docker inspect asa-server-1` - you might need to censor some things, so please go through the output before posting it here (e.g. if you specified the server or rcon password on command line).

Your issue sounds like a routing issue tbh.

**Comment by itsjohnn22** [comment]
So, I'm getting this error in console now. The Compose file './docker-compose.yml' is invalid because:
services.asa-server-1.networks.asa-network contains unsupported option: 'driver_opts'

**Comment by itsjohnn22** [comment]
I fixed it. I can connect by going to single player and using the open command. but cannot connect through the server tab.

**Comment by mschnitzer** [comment]
Yes, and I do believe this is because an attempt to connect through the server browser, will let your client try to establish a connection through your external network interface which is behind a router firewall. This will very likely only work if you do a port forwarding, as the server browser is not aware of the fact that you are actually in the same network like your game server, so it is unaware of your local IP.

If you want to have this working, I'd try to forward ports in your router and make them accessible from the internet. But if your intend is to have this server only for you and your family, then I'd not bother with that and use the `open` command instead.

**Comment by itsjohnn22** [comment]
Okay. I'm sorry to bother you so much. But I'm now getting another error. when i run this command: docker exec asa-server-1 cat server-files/ShooterGame/Saved/Config/WindowsServer/GameUserSettings.ini | grep SessionName. I get this output:
Error response from daemon: Container 63ae83e3afa61ce4f974d97615e7198e6bc1b3b542e5ab507ef07cce9c8a61c9 is not running. This started after restarting the server lol.

**Comment by deronneal0** [comment]
So I got the server up and going but for some reason i cant set a admin and server password. I tried to add it in the gameusersettings and yaml file but it didnt work. I also tried a cluster server but the second cluster never showed up after it was running

**Comment by mschnitzer** [comment]
Could you please open a new issue for your case?
